pkgs/development/libraries/stduuid/fix-version.patch

```diff
@@ -0,0 +1,60 @@
+From c889d9f8b419ced3254ec552c81993bab81fa841 Mon Sep 17 00:00:00 2001
```

Comment:
Use `fetchpatch` instead of vendoring it.

Reply:
Done:
- I'm not sure of how reliable is the url I used as this is from a PR. Maybe there is a better way to get a `.patch` from a PR in github? This seems like it could break any time if the author update his `master` branch? Or am I missing something?
- I squashed my amended my old commit, this is the convention I understood from the contributing guide is ok?

Reply:
Use the long format with git hash.

Reply:
Because the fixed commits are in the `master` branch of not the upstream repo, I copied the patch to my own fork on its dedicated branch and referenced that: https://github.com/OlivierLDff/stduuid/commits/fix-version-1.2.3-cmake/

I'm unsure what the github retention policy, so if the pull request author force pushed his master branch at least everything should still be there. It seems github would have kept the orphan commit accessible by sha, but this is unclear. Tell me if you want me to reference the 2 original commit instead the one in the branch of my fork.

Reply:
Hum, the things are a bit more complicated.
The upstream have seen no updates in two years, and there are useful pull requests from December that weren't merged.

For now your solution is fine.